README.md

```diff
@@ -10,3 +10,20 @@ To build a new Solidus Extension, you can run
 solidus extension my_extension
 ```
 
+after you have globally installed the gem outside of your project folder:
+
+```
+gem install solidus_cmd
```

Comment:
I think this part should be before the instructions to build a new extension.

Aside, not sure if we should encourage users to install gems globally. Personally I prefer to install everything inside a gemset. Maybe could be something like:

> You can install the gem by run..

Reply:
Feel free to improve the docs, I've just taken some hint from [this tutorial](https://blog.botreetechnologies.com/solidus-how-to-install-and-write-extensions-72a1546c083b)